epic-views-firehose/cfn.yaml

```diff
@@ -0,0 +1,90 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Description: 'Kinesis stream for epic view events'
+
+Parameters:
+  App:
```

Comment:
App and Stack are now unused

Reply:
Good shout, I've removed them now @tomrf1